proto/grp.proto

```diff
@@ -17,4 +17,30 @@ message Group {
 
   // True if this group has linked a Github token.
   bool github_linked = 4;
+
+  // Whether new users who sign up with an email at the owned domain
+  // are auto-added to this group.
+  bool auto_populate_from_owned_domain = 5;
```

Comment:
@tylerwilliams I added this back because I figure we'll need to know the state of this bool if we're going to display it as a checkbox in the "edit" mode of the UI

Reply:
you can drive the initial state of the checkbox of whether or not owned_domain is populated